src/dmd/frontend.h

```diff
@@ -336,6 +337,27 @@ class ASTNode : public RootObject
     ASTNode();
 };
 
+struct ParentSymbolAttribute final
+{
+    union
+    {
+    private:
```

Comment:
@MoonlightSentinel : I guess that's a bug ? :D

Reply:
Probably missing call to write the visibility before the anonymous declaration causing the `private` to be printed for the first member.

Reply:
#12845